pallets/summary/Cargo.toml

```diff
@@ -30,6 +30,9 @@ frame-support = {default-features = false, git = "https://github.com/paritytech/
 frame-system = {default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
 sp-staking = {default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
 pallet-session = {default-features = false, features = ["historical"], git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
+pallet-eth-bridge = { default-features = false, path = "../eth-bridge" }
```

Comment:
why do we need it as a main dependency. The idea of using traits was to decouple the dependency between eth-bridge and other pallets